src/main/java/io/codearte/jfairy/producer/util/ZhFairyUtil.java

```diff
@@ -0,0 +1,77 @@
+package io.codearte.jfairy.producer.util;
+
+import io.codearte.jfairy.producer.BaseProducer;
+
+import java.util.Random;
+
+/**
+ * io.codearte.jfairy.producer.util.ZhFairyUtil
+ *
+ * @author lhfcws
+ * @since 2017/3/10
+ */
+public class ZhFairyUtil {
+
+	/**
```

Comment:
You can add a private default constructor to make sure this class is only used to call static methods.